Cargo.toml

```diff
@@ -5,6 +5,8 @@ version = "0.3.0"
 authors = ["Jeremy Soller <jeremy@system76.com>"]
 edition = "2021"
 license = "MIT OR Apache-2.0"
+documentation = "https://pop-os.github.io/cosmic-text/cosmic_text/"
```

Comment:
This doc link is for the master branch, would it be better to link to docs.rs here so the crate references only the documentation of released versions?

Reply:
You're right, fixed.